msteams-platform/m365-apps/cards-loop-component.md

```diff
@@ -9,40 +9,35 @@ ms.date: 01/26/2024
 # Adaptive Card-based Loop components
 
 > [!NOTE]
-> Adaptive Card-based Loop components require Adaptive Cards version 1.6 or later.
+> * Adaptive Card-based Loop components require Adaptive Card schema version 1.6 or later.
+> * Adaptive Card-based Loop components aren't available in Microsoft Teams and Microsoft Outlook on macOS and mobile clients.
 
-Adaptive Card-based Loop components enable you to build collaborative experiences within your Microsoft Teams [message extensions](../messaging-extensions/what-are-messaging-extensions.md) that work across Microsoft 365. Adaptive Card-based Loop components have actionable content that enables users to make live updates without having to switch context between Microsoft 365 apps, such as Teams and Outlook.
-
-[Loop components](https://support.microsoft.com/office/first-things-to-know-about-loop-components-in-microsoft-teams-ee2a584b-5785-4dd6-8a2d-956131a29c81) were first released in Teams followed by other Microsoft 365 apps such as Outlook, Whiteboard, and Loop app. Loop components allow users to collaborate and share live content with others in the chat, email, meeting, Whiteboard or Loop app. Because Loop components stay in sync across different Microsoft 365 apps, they enable users to coauthor content and make real-time updates on their content. For more information, see [overview of Loop components in the Microsoft 365 ecosystem](/microsoft-365/loop/loop-components-teams).
+Adaptive Card-based Loop components enable you to build collaborative experiences within your Teams message extensions that work across Microsoft 365. Adaptive Card-based Loop components have actionable content that enables users to make live updates without having to switch contexts between Microsoft 365 apps, such as Teams and Outlook.
 
 Following is an example of an Adaptive Card-based Loop component:
 
-:::image type="content" source="~/assets/images/adaptive-cards/adaptive-card-loop.png" alt-text="Example of an Adaptive Card Loop component.":::
-
-This article provides an overview of how to build and test your Adaptive Card-based Loop components.
+:::image type="content" source="~/assets/images/adaptive-cards/adaptive-card-loop.png" alt-text="Screenshot shows an Adaptive Card-based Loop component.":::
 
-## Prerequisites
+## Build Adaptive Card-based Loop component
 
-Before you build an Adaptive Card-based Loop component, ensure that you meet the following prerequisites:
+To build an Adaptive Card-based Loop component, follow these steps:
```

Comment:
How about adding a note that there's a readymade sample you can use with this tutorial: https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-unfurling-ac-loop-components/nodejs

Reply:
@erikadoyle, I am not sure how we can "link to the specific code (or add a snippet) of the sample that illustrates each step" as I don't think it's a strict one-on-one mapping. Some of the steps also seem to happen outside an IDE.

Also, given that each step links out to a different page, my guess is the developer is mostly jumping in and out of this page (a necessary evil) and following the procedure in the page that they have jumped out to. This way, this page becomes a waypoint for them in the journey of building the Loop component. I think having code snippets here wouldn't provide much value.

I am just being careful here in order to not complicate the steps. Let me know what you think.